core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2966,6 +2966,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_COPY_FROM_LOCAL_THREADS =
```

Comment:
would this make more sense as a CLI argument to copyFromLocal? That will make it easier to discover and set

Reply:
done

Reply:
nit: can you also update the usage string